build-libs/redirects.js

```diff
@@ -167,20 +168,72 @@ async function getRedirectsForProduct(
 
 	let repo = product
 
-	const rawRedirects = isDeployPreview(product)
```

Comment:
This was flagged by linting as a nested ternary. Have refactored to an `if` block while I'm here 👍